data/language/nl-NL.yml

```
@@ -1695,8 +1695,8 @@ strings:
   1657: Vlak land
   1658: Heuvels
   1659: Bergen
-  1660: Half Mountains, Half Hills
-  1661: Half Mountains, Half Flat
+  1660: "50% bergen, 50% heuvels"
+  1661: "50% bergen, 50% vlak land"
```

Ik ben hier niet dol op, maar kan ook niet echt iets beters bedenken. Misschien het volgende?
```suggestion
  1660: "Afwisselend bergen en heuvels"
  1661: "Afwisselend bergen en vlak land"
```

---

Is ook prima.